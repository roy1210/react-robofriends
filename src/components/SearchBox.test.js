import React from 'react';
import { shallow } from 'enzyme';
import '../setupTests';
import SearchBox from './SearchBox';

it('renders SearchBox without crashing', () => {
  const mockSearch = 'Leanne';
  const components = shallow(<SearchBox searchfield={mockSearch} />);
  expect(components.debug(mockSearch)).toMatchSnapshot();
});
