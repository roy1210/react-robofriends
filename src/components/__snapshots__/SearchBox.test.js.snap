// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders SearchBox without crashing 1`] = `
"<div className=\\"pa2\\">
  <input aria-label=\\"Search Robot\\" className=\\"pa3 ba b--green bg-lightest-blue\\" type=\\"search\\" placeholder=\\"search robots\\" value=\\"Leanne\\" onChange={[undefined]} />
</div>"
`;
